acceptance/framework/config/config.go

```diff
@@ -70,6 +70,9 @@ type TestConfig struct {
 	EnableEnterprise  bool
 	EnterpriseLicense string
 
+	SkipDataDogTests        bool
+	DatadogHelmChartVersion string
```

Comment:
1. I think we can get rid of the `skipDataDogTests` flag altogether - the datadog tests won't run in CI unless we change the YAML we were looking at yesterday.
2. I think there is no way to set `DatadogHelmChartVersion`, so we can just remove it.

Reply:
😮‍💨 my bad, I didn't realize automerge was set.

Reply:
Nah, that's my bad. Sorry 🫤